### PR TITLE
[dist][hotfix] Fix name of Hadoop filesystem plugin 

### LINKED ISSUE #178: [Bug] Lakehouse compaction service can't load hadoop plugin
### Search before asking

- [X] I searched in the [issues](https://github.com/alibaba/fluss/issues) and found nothing similar.

### Fluss version

0.5.0 

### Minimal reproduce step

Fluss use hdfs as remote storage, and start compaction service with `lakehouse.sh`.

### What doesn't meet your expectations?

Can't find the fluss-fs-hadoop plugin although it's in `plugins/hadoop` directory

### Anything else?

When it's hdfs as remote storage, the `lakehouse.sh` will just extract the `hdfs` scheme and try to find the hdfs plugin in directory `plugins/hdfs`, but it's in `plugins/hadoop`. Maybe we need do a mapping to map `hdfs`  to `hadoop`

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by michaelkoepf**
### Purpose

Linked issue: close #178 

### Tests

n/a

### API and Format

n/a

### Documentation

n/a

### CODE REVIEW

File: fluss-dist/src/main/assemblies/plugins.xml [hunk 1]
```diff
@@ -27,10 +27,11 @@
 
     <files>
         <!-- filesystem -->
+        <!-- output directory and plugin JAR should correspond to the file system *schema*, i.e., plugins/<schema>/ and fluss-fs-<schema>-${project.version}.jar -->
         <file>
             <source>../fluss-filesystems/fluss-fs-hadoop/target/fluss-fs-hadoop-${project.version}-with-dependencies.jar</source>
-            <outputDirectory>plugins/hadoop/</outputDirectory>
-            <destName>fluss-fs-hadoop-${project.version}.jar</destName>
+            <outputDirectory>plugins/hdfs/</outputDirectory>
+            <destName>fluss-fs-hdfs-${project.version}.jar</destName>
```
Comment (wuchong) [inline fluss-dist/src/main/assemblies/plugins.xml]: Do we need to rename the jar as well? @luoyuxia

Comment (luoyuxia) [reply fluss-dist/src/main/assemblies/plugins.xml]: It works whether rename or not rename the jar.  But I'd prefer not to rename it sot that we can know the jar is from the jar `fluss-fs-hadoop` built by our fluss-fs-hadoop moudle.

Comment (wuchong) [reply fluss-dist/src/main/assemblies/plugins.xml]: +1

**Review by wuchong — APPROVED** [review]
LGTM

**Pull request merged**